notebooks/tm/hospitals-isochrones/02-contours-to-adminlevel.ipynb

```diff
@@ -0,0 +1,997 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #19.    hospitals_gdf = filtered_doh[filtered_doh["typeofheal"] == "Hospital"]
</pre><p>Add header "Select type of health facility", define variable HEALTH_FACILITY_TYPE = "Hospital"</p><p>Put this line in a new cell</p><pre class="ql-syntax" spellcheck="false"> hospitals_gdf = filtered_doh[filtered_doh["typeofheal"] == HEALTH_FACILITY_TYPE]
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thinkingmachines/lacuna-fund-climate-health/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>I realize I can remove this section! This filtered gdf is not used anywhere else in the notebook. </p>

Reply:
Removed these cells since they were unused in the later parts of the notebook.